uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -687,7 +728,7 @@ define([
             setStatuses();
           }
         } else if (vm.isMode('create')) {
-          vm.leaveRequest.status_id = vm.leaveRequestStatuses['3'].name;
+          vm.leaveRequest.status_id = vm.leaveRequestStatuses['3'].value;
```

Comment:
backend was throwing error as it was expecting integer now while saving, earlier the names were working , hence changed